source/vocab/files-packages-representations.ttl

```diff
@@ -141,3 +142,12 @@
     rdfs:label "height"@en, "höjd"@sv ;
     sdo:domainIncludes :MediaObject ; # TODO: broaden domain later to also apply on Manifest and physical objects
     owl:equivalentProperty sdo:height .
+
+:colorSpace a owl:ObjectProperty ;
+    :category :pending ;
+    rdfs:label "color space"@en, "färgrymd"@sv ;
+    sdo:domainIncludes :MediaObject ;
+    owl:equivalentProperty exif:colorSpace .
+
+:imageBitDepth a owl:DatatypeProperty ;
+  owl:propertyChainAxiom (:digitalCharacteristic [ :isSelfType :ImageBitDepth ] :value) .
```

Comment:
Given that values are extracted from image files, we can also define and use the specific (sub-)property; like:
```suggestion
:imageBitDepth a owl:DatatypeProperty ;
    :category :pending ;
    owl:propertyChainAxiom (:digitalCharacteristic [ :isSelfType :ImageBitDepth ] :value) .

:bitsPerSample a owl:DatatypeProperty ;
    :category :pending ;
    sdo:domainIncludes :MediaObject ;
    rdfs:subPropertyOf :imageBitDepth , exif:bitsPerSample .
 ```

Reply:
The upside is a clear mapping to EXIF. The downside is more specificity equals more choice (which increases the risk of arbitrary differences in detail). There are ways of managing the "difference creep" (using property categories, or more validation).